proxy/coco/tests/replication.rs

```diff
@@ -215,3 +247,75 @@ async fn can_fetch_project_changes() -> Result<(), Box<dyn std::error::Error>> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn can_sync_on_startup() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
How do you feel about using the `testnet` code here instead?

Reply:
Let's address that in #908 - I fear it might introduce too much noise in this PR which already is not that lightweight.

Reply:
Well what I mean is do the same style as: https://github.com/radicle-dev/radicle-upstream/blob/fintan/testnetz/proxy/coco/tests/replication.rs#L14-L74

And I'm just suggesting this test in particular since it's being introduced fresh anyway. That way we don't need to go back and change it later.

Feel free to resolve this if you'd still rather not.

Reply:
Yeah, as much as I agree with the approach -- it will divert from this change-set. Already got feedback that this PR is largish and not easy to review. We should feel safe with #908 being there to keep us honest.

Reply:
:+1: